.eslintrc.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     'no-throw-literal': 'error',
     'no-useless-escape': 'off', // TODO: bring this back
     'no-var': 'error',
+    'no-prototype-builtins': 'off',
```

Comment:
@locks can you make an issue to remove this (seems good for a "help wanted" type of thing)?

Reply:
https://github.com/emberjs/ember.js/issues/18914